lib/modules/manager/helmfile/extract.ts

```diff
@@ -130,7 +138,11 @@ export async function extractPackageFile(
       const repository = doc.repositories?.find(
         (repo) => repo.name === repoName,
       );
-      if (repository?.oci) {
+      if (isOciUrl(dep.chart)) {
+        res.datasource = DockerDatasource.id;
+        res.packageName = repoName + '/' + depName;
```

Comment:
```suggestion
        res.packageName = dep.chart;
```
it should probably use registry aliases, like below

Reply:
I tested against a test repository and it won't work with this suggestion, as the packageName must be the full name.

The registry aliases never make sense with an oci url, as it is already the complete url/name.